draft-ietf-dnsop-svcb-https.md

```diff
@@ -245,11 +245,15 @@ appear in all capitals, as shown here.
 
 The SVCB DNS resource record (RR) type (RR type ???)
 is used to locate endpoints that can service an origin.
-There is special handling for the case of "https" origins.
 
 The algorithm for resolving SVCB records and associated
 address records is specified in {{client-behavior}}.
 
+Other resource record types in the same form as SVCB
```

Comment:
We defined the term "SVCB-compatible" for this purpose in the terminology section.  I'd prefer to maintain consistency across the document if possible, e.g.
```suggestion
Other SVCB-compatible resource record types
```